.gitignore

```diff
@@ -11,3 +11,10 @@ docs/_build
 regparser.egg-info/
 
 .eregs_index/
+
+# eregs output
+/fr_cache.sqlite
+/diff/
+/layer/
+/notice/
+/regulation/
```

Comment:
I agree with fr_cache.sqlite, but if you're outputting the diff/layer/notice/regulation directories in `.`, you've probably configured something wrong. They probably shouldn't be ignored